src/webrtc-wowza-player.ts

```diff
@@ -239,7 +239,9 @@ export class WowzaWebRTCPlayer extends EventEmitter {
         this.video.srcObject = stream;
       }
     } catch (error) {
-      this.video.src = window.URL.createObjectURL(stream);
+      if (stream instanceof MediaSource) {
+        this.video.src = window.URL.createObjectURL(stream);
+      }
```

Comment:
@vthibault I have a doubt on this solution (only way I made it work) : is there a risk it breaks / crashes ? 

Problem is : createObjectURL is expecting a `MediaSource | Blob` type, but `stream` is a `MediaStream` and is missing some of the MediaSource properties. Unfortunately, `(stream as MediaSource | Blob)` doesn't seem to work